src/main/kotlin/org/jitsi/nlj/dtls/DtlsUtils.kt

```diff
@@ -84,7 +84,7 @@ class DtlsUtils {
          */
         fun chooseSrtpProtectionProfile(ours: IntArray, theirs: IntArray): Int {
             return try {
-                theirs.first(ours::contains)
+                ours.first(theirs::contains)
```

Comment:
This was intentional to let us dictate the priority, yeah?

Reply:
Yes, that's the idea.